### PR TITLE
chore: add fields validations in schema

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: django_forest/utils/schema/validations.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        'message': 'Ensure this value is greater than or equal to %(limit_value)s characters'
+    },
+}
+
+
+def handle_validator(validator, f):
+    if isinstance(validator, MaxLengthValidator) \
+            or isinstance(validator, MinLengthValidator) \
+            or isinstance(validator, MaxValueValidator) \
+            or isinstance(validator, MinValueValidator):
```
Comment (guillaumedeslandes) [inline django_forest/utils/schema/validations.py:29]: Do you need to check against classes, or is this enough?
```suggestion
    if validator.code in VALIDATORS:
```

File: django_forest/utils/schema/validations.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        v = VALIDATORS[validator.code]
+        f['validations'].append({
+            'type': v['type'],
+            'message': v['message'] % {'limit_value': validator.limit_value},
+            'value': validator.limit_value
+        })
+    elif isinstance(validator, RegexValidator):
```
Comment (guillaumedeslandes) [inline django_forest/utils/schema/validations.py:36]: If change above is accepted, maybe check against code here to be consistent.

File: django_forest/utils/schema/validations.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    return f
+
+
+def handle_validations(field, f):
+    if not field.is_relation and not field.auto_created:
+        f = handle_validators(field.validators, f)
+        f = handle_is_present(field, f)
+
+    if len(f['validations']) == 0:
+        del f['validations']
```
Comment (guillaumedeslandes) [inline django_forest/utils/schema/validations.py:72]: If this necessary? I do not see where the `validations` field is added.

Comment (GuillaumeCisco) [reply django_forest/utils/schema/validations.py:72]: It is by default added, thanks to the `get_default` function.
This is what `forest-rails` do too... I didn't think much in this case...
Best should be to add `validation` when needed indeed, and not deleting it when not needed...

File: django_forest/utils/schema/validations.py (hunk 2 above)
Comment (GuillaumeCisco) [reply django_forest/utils/schema/validations.py:36]: Hum, I've just experimented in regex case, the validator has no code. This is not really true, in fact, the code is `invalid` which is not something I'd like to check against...
Here is the definition for more insight:
```python
@deconstructible
class RegexValidator:
    regex = ''
    message = _('Enter a valid value.')
    code = 'invalid'
    inverse_match = False
    flags = 0
```
I think keeping the `isinstance` check is the best for now

**Pull request merged**